### ISSUE TITLE
[vizbuilder] add indentation to depth selector (show it as a child of dimension selector)

### ISSUE THREAD

**Issue by cnavarreteliz** [Datawheel/canon#150]
![image](https://user-images.githubusercontent.com/5233593/44994654-8751af00-af97-11e8-86ef-0b1e8aeec4a4.png)

**Comment by frabarz** [comment]
I implemented it in 6cb1316840633108c99c47439d0a24b649b0c18d, but since this is more CSS related, I just changed the HTML structure to style it easier. I'm not sure if there's another kind of changes I should do here.

**Comment by davelandry** [comment]
@frabarz makes sense. for now, can you also put in a default `margin-left` for the depth selector?
